pkgs/tools/filesystems/zxfer/default.nix

```
@@ -0,0 +1,41 @@
+{ lib, stdenv, fetchFromGitHub
+, coreutils, gnused, gawk, gnugrep
+, openssh, rsync
+, gzip }:
+
+stdenv.mkDerivation rec {
+  pname = "zxfer";
+  version = "1.1.7";
+
+  src = fetchFromGitHub {
+    sha256 = "11SQJcD3GqPYBIgaycyKkc62/diVKPuuj2Or97j+NZY=";
+    rev = "v${version}";
+    repo = "zxfer";
+    owner = "allanjude";
```

Sticking to the most common ordering, makes spotting issues easier.

```suggestion
    repo = "zxfer";
    owner = "allanjude";
    rev = "v${version}";
    sha256 = "11SQJcD3GqPYBIgaycyKkc62/diVKPuuj2Or97j+NZY=";
```